portfolio/src/main/webapp/index.html

```diff
@@ -23,18 +23,20 @@
       <input type="submit" class="submit" value="Show"></input>
     </form>
     <form id="sortComments">
-      <label for="sortingMethods">Sort:</label>
-      <select id="sortingMethods">
-        <option value="time">Time (newest)</option>
-        <option value="likes">Time (oldest)</option>
+      <label for="sortMethod">Sort:</label>
+      <select id="sortMethod" onchange="applySortMethod()">
+        <option value="DESCENDING" selected="selected">Time (newest)</option>
```

Comment:
these shouldn't be all-caps.  Using capitalization in general is asking for trouble because then it raises the question of when capitalization matters and if so what the impact of that is.

Reply:
Noted; changed to lower case bools

Reply:
bool strings